features/step_definitions/chart_of_accounts/account.rb

```diff
@@ -486,3 +486,94 @@
       pending "I don't know how to remember a \" #{target} \" Account."
   end
 end
+
+And /^I create an Account using a CG account with a CG Account Responsibility ID in range (.*) to (.*)$/ do |lower_limit, upper_limit|
+  #method call should return an array
+  account_numbers = find_cg_accounts_in_cg_responsibility_range(lower_limit, upper_limit)
+
+  options = {
+      chart_code:               get_aft_parameter_value(ParameterConstants::DEFAULT_CHART_CODE),
+      account_number:           account_numbers.sample,
+      name:                     'Test create Account with CG account',
+  }
+  @account = create AccountObject, options
+  @account.icr_accounts.updates_pulled_from_page :old
+  @account.icr_accounts.updates_pulled_from_page :new
```

Comment:
Why are you pulling both the new and old ICR Accounts into the same Account object?